model-optimizer/extensions/front/tf/CTCGreedyDecoderReplacement.py

```diff
@@ -0,0 +1,145 @@
+# Copyright (C) 2018-2021 Intel Corporation
```

Comment:
please use git mv for moving files. Otherwise, we have to review it from scratch.

Reply:
Looks like it is github problem and it shows moved files as deleted and created in new place.

Reply:
ok